BUILDING.md

```diff
@@ -84,3 +84,48 @@ To build the function examples under `./examples` you will need to add the WASM
 ```shell
 rustup target add wasm32-unknown-unknown
 ```
+
+## Jetson Nano/Ubuntu 18.04
```

Comment:
Is that really still 18.04? That is EOL. Would prefer if we would target more modern systems only.

Reply:
Unfortunately, yes. But that is NOT a target of the project, just a board I borrowed from a colleague.  So there's no need to support it officially.